modules/launching-your-first-operator.adoc

```diff
@@ -0,0 +1,204 @@
+navigation.
++
+[NOTE]
+====
+If you want to enable additional users to view, create, and manage Subscriptions
+in a project, they must have the `admin` and `view` roles for that project, as
+well as the `view` role for the `operator-lifecycle-manager` project. Cluster
+administrators can add these roles using the following commands:
+
+----
+$ oc policy add-role-to-user admin <user> -n <target_project>
+$ oc policy add-role-to-user view <user> -n <target_project>
+$ oc policy add-role-to-user view <user> -n operator-lifecycle-manager
+----
+
+This experience will be simplified in future releases of the OLM.
+====
+
+. Subscribe the desired project to the Couchbase catalog source from either the
+web console or CLI.
++
+Choose one of the following methods:
++
+--
+- For the web console method, ensure you are viewing the desired project, then
+click *Create Subscription* on an Operator from this screen to install it to the
+project.
+
+- For the CLI method, create a YAML file using the following definition:
++
+.*_couchbase-subscription.yaml_* file
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  generateName: couchbase-enterprise-
+  namespace: couchbase-test <1>
+spec:
+  source: certified-operators
+  name: couchbase-enterprise
+  startingCSV: couchbase-operator.v1.0.0
+  channel: preview
+----
+<1> Ensure the `namespace` field in the `metadata` section is set to the desired
+project.
++
+Then create the Subscription using the CLI:
++
+----
+$ oc create -f couchbase-subscription.yaml -n couchbase-test
+----
+--
+
+. After the Subscription is created, the Operator then appears in the *Cluster Service Versions*
+screen, which is the catalog users can use to launch the software provided by
+the Operator. Click on the Couchbase Operator to view more details about this
+Operator's features:
++
+.Couchbase Operator overview
+image::couchbase-operator-overview.png[]
+
+. Before creating the Couchbase cluster, create a secret with the following
+definition using the web console or CLI that holds credentials for the super
+user account. The Operator reads this upon start up and configures the database
+with these details:
++
+--
+.Couchbase secret
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: couchbase-admin-creds
+  namespace: couchbase-test <1>
+type: Opaque
+stringData:
+  username: admin
+  password: password
+----
+<1> Ensure the `namespace` field in the `metadata` section is set to the desired
+project.
+--
++
+Choose one of the following methods:
++
+--
+- For the web console method, click *Workloads* -> *Secrets* from the left
+navigation, then click *Create* and choose *Secret from YAML* to enter the
+secret definition.
+
+- For the CLI method, save the secret definition to a YAML file (for example, *_couchbase-secret.yaml_*) and use the CLI to create it in the desired project:
++
+----
+$ oc create -f couchbase-secret.yaml -n couchbase-test
+----
+--
+
+. Create the new Couchbase cluster.
++
+[NOTE]
```

Comment:
The line 127 ~ 139 is unnecessary since we already grant the `admin` role to the general user. So, I suggest removing them.

Reply:
@jianzhangbjz Are you referring to the `oc policy` steps from lines 32 ~ 40? I phrased that [NOTE] box as optional, if they (optionally) wanted another user that could create Subscriptions. But this line 127 ~ 139 is just for if they (optionally) wanted another user that could manage Instances (but not have the fuller Subscription-level permissions).

Reply:
OK, I see, thanks!